src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -810,8 +811,14 @@ export default {
 
 		handleRetry() {
 			if (this.sendingErrorCanRetry) {
-				EventBus.$emit('retry-message', this.id)
-				EventBus.$emit('focus-chat-input')
+				if (this.sendingFailure === 'failed-upload') {
+					const caption = this.renderedMessage !== this.message ? this.message : undefined
+					this.$store.dispatch('retryUploadFiles', { uploadId: this.messageObject.uploadId, caption })
+					this.$store.dispatch('removeTemporaryMessageFromStore', this.messageObject)
```

Comment:
It would be better if it is in `retryUploadFiles` action and also remove all files from that uploadId not only the one that has the button clicked.

Reply:
Yeah, I need to test it with multiple files upload